complaint_search/es_interface.py

```diff
@@ -59,25 +61,38 @@ def get_meta():
 # - submitted_via: filters a list of ways the complaint was submitted
 # - tag - filters a list of tags
 def search(**kwargs):
-
+    params = copy.deepcopy(PARAMS)
+    params.update(**kwargs)
     search_builder = SearchBuilder()
-    search_builder.add(**kwargs)
+    search_builder.add(**params)
     body = search_builder.build()
 
     post_filter_builder = PostFilterBuilder()
-    post_filter_builder.add(**kwargs)
+    post_filter_builder.add(**params)
     body["post_filter"] = post_filter_builder.build()
 
     # format
     res = None
-    format = kwargs.get("format", "json")
+    format = params.get("format")
     if format == "json":
-        if not kwargs.get("no_aggs", False):
+        if not params.get("no_aggs"):
             aggregation_builder = AggregationBuilder()
-            aggregation_builder.add(**kwargs)
+            aggregation_builder.add(**params)
             body["aggs"] = aggregation_builder.build()
 
-        res = get_es().search(index=_COMPLAINT_ES_INDEX, doc_type=_COMPLAINT_DOC_TYPE, body=body, scroll="10m")
+        res = get_es().search(index=_COMPLAINT_ES_INDEX, 
+            doc_type=_COMPLAINT_DOC_TYPE, 
+            body=body, 
+            scroll="10m")
+
+        num_of_scroll = params.get("frm") / params.get("size")
```

Comment:
Wow, is there no way to scroll ahead other than N pulls?  Granted, most times this will not be called more than once, but I can see someone calling page 100 and having to issue 100 calls to ES